source/config/configFlags.py

```diff
@@ -97,6 +97,23 @@ def _displayStringLabels(self):
 		}
 
 
+@unique
+class BrailleMode(DisplayStringStrEnum):
+	"""Enumeration containing the possible config values for "Braille mode" option in braille settings.
+	Use BrailleMode.MEMBER.value to compare with the config;
+	use BrailleMode.MEMBER.displayString in the UI for a translatable description of this member.
+	"""
+	FOLLOW_CURSORS = "followCursors"
+	SPEECH_EMULATION = 'speechEmulation'
+
+	@property
+	def _displayStringLabels(self):
+		return{
+			# Translators: The label for a braille mode
+			BrailleMode.FOLLOW_CURSORS: _('follow cursors'),
+			# Translators: The label for a braille mode
+			BrailleMode.SPEECH_EMULATION: _('speech emulation')
+		}
```

Comment:
Please ensure there are 2 new lines between class definitions

```suggestion
		}


```